radioapp/ui/home.py

```diff
@@ -1,69 +1,85 @@
-
-    def update_channel(self, channel):
-        self.channel.set(channel)
-
-    def __clock(self):
-        timeNow = time.strftime('%H:%M:%S')
-        self.time.set(timeNow)
-        self.frame.after(200, func=self.__clock)
-        
-    def button1(self):
-        self.b1Action()
-
-    def button2(self):
-        pass
-    
-    def button3(self):
-        pass
+
+class UIHome:
+    """Creates UI Home Screen."""
+
+    def __init__(self, window, master):
+        """Creates a frame, and customizes and adds labels for UI Home Screen."""
+
+        window.title("Home")
+        self.my_frame = tk.Frame(window)
+
+        self.add_top()
+        self.add_middle()
+        self.add_buttons(master)
+
+    def pack_screen(self):
+        """Puts the screen on the main window."""
+
+        self.my_frame.pack(expand=1, fill="both")
+
+    def destroy_screen(self):
+        """Destroys the screen from the main window."""
+
+        self.my_frame.destroy()
+
+    def add_buttons(self, master):
+        """Adds buttons at the bottom frame."""
+
+        self.bottom_frame = tk.Frame(self.my_frame, bg="yellow")
+        self.bottom_frame.pack(side="bottom", fill="x")
+
+        self.button_1 = tk.Button(self.bottom_frame, text="1", bg="#3399ff", fg="white", height=1, width=6)
+        self.button_1.grid(row=7, column=0, sticky="nsew")
+
+        self.button_2 = tk.Button(self.bottom_frame, text="2", height=1, width=6, fg="white", bg="#3399ff")
+        self.button_2.grid(row=7, column=1, sticky="nsew")
+
+        self.button_3 = tk.Button(self.bottom_frame, text="3", height=1, width=7, fg="white", bg="#3399ff")
+        self.button_3.grid(row=7, column=2, sticky="nsew")
+
+    def add_middle(self):
+        """Customizes and adds labels at the middle frame."""
+
+        self.middle_frame = tk.Frame(self.my_frame, bg="#3399ff")
+        self.middle_frame.pack(expand=1, fill="both")
+
+        self.label_4 = tk.Label(self.middle_frame, text="", fg="white", bg="#3399ff", font="Helvetica 9 bold")
+        self.label_4.grid(row=2, column=0, sticky="nsew")
+
+        self.label_5 = tk.Label(self.middle_frame, text=" Channel 0", fg="white", bg="#3399ff", font="Helvetica 9 bold")
+        self.label_5.grid(row=4, column=0, sticky="nsew")
+
+        self.label_6 = tk.Label(self.middle_frame, text="          ", fg="white", bg="#3399ff", font="Helvetica 9 bold")
+        self.label_6.grid(row=4, column=1, sticky="nsew")
+
+        self.label_7 = tk.Label(self.middle_frame, text="-120 dBm", fg="white", bg="#3399ff", font="Helvetica 9 bold")
+        self.label_7.grid(row=4, column=2, sticky="nsew")
+
+    def add_top(self):
+        """Customizes and adds labels at the top frame."""
+
+        self.top_frame = tk.Frame(self.my_frame, bg="yellow")
+        self.top_frame.pack(side="top", fill="x")
+
+        self.label_1 = tk.Label(self.top_frame, text="12:00:00", height=1, width=8, fg="white", bg="#3399ff",
```

Comment:
This button widths result in buttons too big for the window (at least on the version of tkinter that I have installed). I played around with a bit and 3 seems to be good value (though they end up a bit off center)

Reply:
Are you talking about the buttons 1, 2 and 3 or the labels? Could you post a screen shot so I could get a better picture?

Reply:
I see the buttons 1,2,3 being stretched

Reply:
Maybe you need to update your tkinter or python. It's hard to fix when it looks fine on my side, but looks different on yours. My python version is 3.7, and my tkinter version is 8.6. Here is how it looks on mine. 
![home screen](https://user-images.githubusercontent.com/13712591/52157501-05a0a300-2655-11e9-9220-900b6dd299f5.png)

Reply:
The most notable difference here is that @jackdweber and I have both been testing on Linux desktops and this screenshot is from Windows. I'd suggest looking at this on a Linux desktop (since that's closer to what we'll be running.

Reply:
Fixed.